.github/workflows/image.yml

```diff
@@ -0,0 +1,73 @@
+
+name: Push Docker Image
+run-name: PushI Docker Image by @${{ github.actor }}
+
+on: 
+  workflow_dispatch:
+
+  release:
+    types: 
+      - published
+
+permissions:
+  packages: write
+
+# ------------------------------------------------------------------------------------------------------------------------
+env:
+  ACR_NAME: 'ghcr.io'
+  ACR_APP_NAME: 'aik8s_troubleshoot'
+
+# ------------------------------------------------------------------------------------------------------------------------
+jobs:
+  build-and-push-image:
+    runs-on: ubuntu-latest
+    permissions: 
+        packages: write
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4.1.1
+
+      - name: Get repo name
+        id: set-repo
+        run: echo "REPO_NAME=$(basename ${{ github.repository }})" >> $GITHUB_OUTPUT
+
+      # Uses the `docker/login-action` action to log in to the Container registry using the account and password that 
+      # will publish the packages. Once published, the packages are scoped to the account defined here.
+      - name: Log in to the Container registry
+        uses: docker/login-action@327cd5a69de6c009b9ce71bce8395f28e651bf99
+        with:
+          registry: ${{ env.ACR_NAME }}
+          username: ${{ secrets.PUBLISH_PACKAGE_USER }}
```

Comment:
Is this something on your fork? I am not sure we will be able to add secrets for now. Do you mind keep it in the fork for a bit longer and not in the PR?

Reply:
I don't think it needs secrets I didn't have to add anything to my repo

Reply:
ok, let me update it, I think I need to update to Github container registry

https://github.com/docker/login-action?tab=readme-ov-file#github-container-registry

Reply:
see a test run here: https://github.com/karpikpl/TMP-ApplicationInsights-Kubernetes/actions/runs/14607176595